### ISSUE TITLE
[BUG] Accessibility-related issues

### ISSUE THREAD

**Issue by CodingWonders** [CodingWonders/DISMTools#85]
**What is the bug about?**
The program is a bit accessible but could be improved in that area.

**How does it happen?**
Please provide the steps necessary to reproduce this bug:

No steps required

**Expected behavior**

The program should behave nicely with screen readers like Narrator or [Non Visual Desktop Access (NVDA)](https://www.nvaccess.org/download/).

**Screenshots**

No screenshots required

**Additional context**

Affected programs: screen readers (Narrator, NVDA...)

**Comment by CodingWonders** [comment]
This issue will be worked on later
